server/lib/pbench/mock.py

```diff
@@ -0,0 +1,221 @@
+"""
+Module for mocking out behaviors of V1 Elasticsearch.
+"""
+
+import json
+from collections import Counter
+
+
+class MockElasticsearch(object):
+    """A simple mock for the V1 Elasticsearch client object. We really just
+    duck-type it, only providing names for attributes we use.  It is not complete
+    by any stretch of the imagination.
+    """
+    def __init__(self, hosts, max_retries=None):
+        self.hosts = hosts
+        self.max_retries = max_retries
+        self.mpt = _MockPutTemplate()
+        self.msb = _MockStreamingBulk(15, self.mpt)
```

Comment:
Sure.

Reply:
Added a comment but did not add a meaningful name.  It is not the kind of constant we want to change because it impacts the unit test output (explained in the comment).  Is that sufficient?